meta-balena-common/recipes-support/hostapp-update-hooks/hostapp-update-hooks.bb

```diff
@@ -9,6 +9,7 @@ inherit allarch
 
 HOSTAPP_HOOKS = " \
     0-bootfiles \
+    60-balena_migrate_storage \
```

Comment:
need to add all the files and the directory name separately. See https://github.com/balena-os/meta-balena/commit/962e526b2582d66ea4ad15433a45445e1ee9327f#diff-504d12ea9d623c9bcfafcd9e84037198 for an example.